### PR TITLE
Init action

### PR DESCRIPTION
**Pull request by jirimakarius**

### CODE REVIEW

**Review by vladmalynych — APPROVED** [review]
Good job👍

File: lib/check.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    checkWorkingHours(date) {
+        const day = this.dayFromSchedule(date)
+
+        if (day===undefined || day.startHour === undefined || day.endHour === undefined) {
+            return false
+        }
+
+        if (day.startHour > date.hour || date.hour >= day.endHour) {
+            return false
+        }
```
Comment (petr-lessner) [inline lib/check.js]: This is easier to read.
```suggestion
        if (date.hour < day.startHour || date.hour >= day.endHour) {
            return false
        }
```

**Review by petr-lessner — APPROVED** [review]
LGTM

Typing would be nice.
Please add newlines at end of files.

File: lib/check.js (hunk 1 above)
Comment (petr-lessner) [reply lib/check.js]: Also the naming is little bit confusing. What is `day` and `date`? It could be `now` and `schedule_day`.

**Pull request merged**